src/Controller/StatusController.php

```diff
@@ -31,6 +41,57 @@ public function statusAction(Request $request)
         return $this->render('@Clifton/Status/status.html.twig', [
             'now' => new DateTime(),
             'dbConnectivity' => $dbalConnection->isConnected() || $dbalConnection->connect(),
+            'dbErrorCheck' => $hasDBIssues,
         ]);
     }
+
+    /**
+     * This insanity is copy/pasted from Faucet. It detects whether an Exception indicates that the
+     * database is down. No, there is no single exception for that.
+     * When the database is down we return a 200 status. Other DB exceptions return a 500 status.
+     * See programmes ticket https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5534
+     */
```

Comment:
Can we drop a little comment in there that says what returning true / false mean? It looks like true is a DB issue that we care about